aea/cli_gui/aea_cli_rest.yaml

```diff
@@ -133,8 +133,8 @@ paths:
           schema:
             type: string
 
-  /agent/{agent_id}/{item_type}/{item_id}:
-    delete:
+  /agent/{agent_id}/remove/{item_type}:
```

Comment:
This path is inconsistent with say the scaffold command where it is `/agent/{agent_id}/{item_type}/scaffold`